node/packages/aws-lambda-sdk/instrument/index.js

```diff
@@ -28,6 +28,29 @@ const { awsLambda: awsLambdaSpan, awsLambdaInitialization: awsLambdaInitializati
 
 const toProtobufEpochTimestamp = awsLambdaSpan.constructor._toProtobufEpochTimestamp;
 
+const resolveBodyString = (data, prefix) => {
+  if (data === undefined) return null;
+  data = { ...data };
```

Comment:
```suggestion
```
Do we need to do a spread here?

Reply:
Indeed more optimal would be to make it only when we strip the binary body (and in that case is needed, as the same object is processed by user code). Updated